### ISSUE TITLE
"huh?" in workflow config - namespace doc

### ISSUE THREAD

**Issue by MetRonnie** [cylc/cylc-flow#4208]
`flow.cylc[runtime][<namespace>]`

https://github.com/cylc/cylc-flow/blob/3537c161be43ddae7a1aa5373de14445ce159074/cylc/flow/cfgspec/workflow.py#L659-L663

**Comment by hjoliver** [comment]
Text commented out and "-huh?" added here:
```
commit bb13110de9f8c99bd6c48d3d8bd3aec27310a201 (HEAD)
Author: Hilary James Oliver <h.oliver@niwa.co.nz>
Date:   Thu Feb 7 21:54:13 2013 +1300

    Updated the user guide for MI.
```

**Comment by hjoliver** [comment]
So evidently I thought it was weird back in 2013, and commented it out.  Somehow it got un-commented again later, probably on transfer to the new self-documenting conf spec structure.  So, I think we can just remove that and say `.` is not allowed because it is  the cycle point delimiter.

**Comment by MetRonnie** [comment]
Might be possible to use the unicode validator autodocumenting feature

**Comment by wxtim** [comment]
Closed by merger of PR #4290
